docs/apps/lnw/es2k/es2k-linux-networking-lag.md

```diff
@@ -0,0 +1,336 @@
+p4rt-ctl add-entry br0 linux_networking_control.handle_rx_from_wire_to_ovs_table \
+    "vmeta.common.port_id=1,user_meta.cmeta.bit32_zeros=0,action=linux_networking_control.set_dest(27)"
+
+# Configuration for control packets between underlay IDPF netdev VSI-11 to physical port 1
+p4rt-ctl add-entry br0 linux_networking_control.handle_tx_from_host_to_ovs_and_ovs_to_wire_table \
+    "vmeta.common.vsi=11,user_meta.cmeta.bit32_zeros=0,action=linux_networking_control.set_dest(1)"
+```
+
+### LAG configuration
+
+Create a LAG interface and add 2 IDPF netdevs as LAG members in it. Assign IP to the LAG interface.
+
+Below configuration assumes
+
+- Underlay IDPF netdev has a VSI value 10 for first LAG member
+- Underlay IDPF netdev has a VSI value 11 for second LAG member
+
+```bash
+ip link add bond0 type bond miimon 100 mode active-backup
+ip link set <IDPF netdev for VSI 10> down
+ip link set <IDPF netdev for VSI 10> master bond0
+ip link set <IDPF netdev for VSI 11> down
+ip link set <IDPF netdev for VSI 11> master bond0
+ip link set bond0 up
+ifconfig bond0 40.1.1.1/24 up
+ip route change 40.1.1.0/24 via 40.1.1.2 dev bond0
+```
+
+Sample link partner underlay configuration.
+Create a LAG interface and assign IP to the LAG interface
+Assuming ens785f0np0 and ens785f1np1 ports are B2B connected to MEV.
+
+```bash
+ip link add bond0 type bond miimon 100 mode active-backup
+ip link set ens785f0np0 down
+ip link set ens785f0np0 master bond0
+ip link set ens785f1np1 down
+ip link set ens785f1np1 master bond0
+ip link set bond0 up
+ifconfig bond0 40.1.1.2/24 up
+
+ip link add vxlan0 type vxlan id 0 dstport 4789 remote 40.1.1.1 local 40.1.1.2 dev bond0
+ip addr add 40.1.1.2/24 dev bond0
+ip addr add 99.0.0.3/24 dev vxlan0
+ip link set vxlan0 up
+ip link set bond0 up
+```
+
+### Test the ping scenarios
+
+- Verify the underlay ping is working fine via the active path.
+- Verify the overlay ping between VM's on same host is working fine via the active path.
+- Note the active link by using the command "cat /proc/net/bonding/bond0" on both MEV host and LP device.
+- Shut the active link on both MEV host and its peer Link Partner.
+- Verify the switchover happened from active to backup interface.
+- Verify both underlay and overlay ping are working fine and taking a backup path.
+
+## Limitations
+
+Current Linux Networking support for the networking recipe has following limitations:
+
+- All VLAN interfaces created on top of IDPF netdev, should always be in lowercase format "vlan+vlan_id"
+Ex: vlan1, vlan2, vlan3 ... vlan4094.
+- Set the pipeline before adding br-int port, vxlan0 port, and adding vlan ports to br-int bridge.
+- VxLAN destination port should always be standard port. i.e., 4789. (limitation by p4 parser)
+- We do not support any ofproto rules that would prevent FDB learning on OvS.
+- VLAN Tagged packets are not supported.
+- For VxLAN tunneled packets only IPv4-in-IPv4 is supported.
```

Comment:
Do we need to say anything here about IPv6 in the limitations section?
There is no mention of IPv6 in whole document

Reply:
I don't think it is required.